netsim/augment/links.py

```diff
@@ -1077,15 +1077,11 @@ def copy_link_gateway(link: Box, nodes: Box) -> None:
         intf.gateway[af] = link.gateway[af]
 
 """
-Set node.af flags to indicate that the node has IPv4 and/or IPv6 address family configured
+Set node.af flags to indicate that the node has IPv4 and/or IPv6 address family configured on its interfaces
 """
 def set_node_af(nodes: Box) -> None:
   for n in nodes.values():
     for af in ['ipv4','ipv6']:
```

Comment:
```for af in ['ipv4','ipv6'] if af not in n.af```

Reply:
Yeah, we could do that optimization. Won't buy us much, but it's nice to have ;)